hibp.js

```diff
@@ -58,7 +58,19 @@ const HIBP = {
   },
 
   async getUnsafeBreachesForEmail(sha1, allBreaches) {
-    return await this.getBreachesForEmail(sha1, allBreaches, true);
+    const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
+
+    const breachLists = {
+      "websiteBreaches": allFoundBreaches.filter(breach => breach.IsSpamList === false),
+      "spamLists": allFoundBreaches.filter(breach => breach.IsSpamList === true),
+    };
+
+    if (breachLists.websiteBreaches.length === 0 && breachLists.spamLists.length === 0) {
+      return false;
```

Comment:
Not sure if this is weird to have a method return an Object or Boolean. Or if we just return an empty object or something. I guess it depends on how it’s used.

Reply:
An empty object is a truthy value. :/ Since we're using this return value in `if` checks I'd like to keep this for now. Adding something like Flow to the project would help with this longer-term.